.github/workflows/build.yml

```
@@ -22,16 +22,12 @@ env:
   MAVEN_USERNAME: ${{ secrets.NEXUS_USERNAME }}
   MAVEN_CLI_OPTS: >-
     --settings settings.xml
-    -fae -U -B -V
-    -Dspring.main.banner-mode=off
-    -Ddoclint=none
-    -DaltReleaseDeploymentRepository=alfresco-public::default::https://artifacts.alfresco.com/nexus/content/repositories/releases
-    -DaltSnapshotDeploymentRepository=alfresco-public-snapshots::default::https://artifacts.alfresco.com/nexus/content/repositories/snapshots
+    -fae -U -V --no-transfer-progress -Dspring.main.banner-mode=off -Ddoclint=none
```

```suggestion
    -V --no-transfer-progress -Dspring.main.banner-mode=off -Ddoclint=none
```

`-fae` (fail at end) should not be needed here: this is useful when you want to go through all the tests that you can go to, here we want to fail fast.

`U` (forcing snapshots update) should not be needed and could slow down the build.